src/lib/Scenes/Home/Components/Sales/Components/ZeroState/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { WebView } from "react-native"
+
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+
+export class ZeroState extends React.Component<null> {
+  render() {
+    return (
+      <WebView
+        onShouldStartLoadWithRequest={e => {
+          if (e.navigationType === "click") {
+            SwitchBoard.presentNavigationViewController(this, e.url)
+            return false
+          }
+          return true
+        }}
+        source={require("./index.html")}
```

Comment:
I wonder if this will work once inside eigen and not using the dev build folders. 🤷🏽‍♂️

worth a check once it's shipped

Reply:
Hmm, yeah def need to check that

Reply:
I'll double check but if it doesn't we can embed the HTML in the `.tsx` file instead